ci/travis/install-dependencies.sh

```diff
@@ -32,14 +32,28 @@ fi
 # Upgrade pip and other packages to avoid incompatibility ERRORS.
 pip install --upgrade pip # setuptools cloudpickle urllib3
 
+# If we're in a CI environment, do some configuration
+if [ "${TRAVIS-}" = true ] || [ -n "${GITHUB_TOKEN-}" ]; then
+  pip config --user set global.disable-pip-version-check True
+  pip config --user set global.no-color True
+  pip config --user set global.progress_bar off
+  pip config --user set global.quiet True
+fi
+
 if [[ "$PYTHON" == "3.6" ]] && [[ "$platform" == "linux" ]]; then
   sudo apt-get update
-  sudo apt-get install -y python-dev python-numpy build-essential curl unzip tmux gdb
+  sudo apt-get install -y python-dev python-numpy build-essential curl unzip tmux gdb libunwind-dev
   # Install miniconda.
   wget -q https://repo.continuum.io/miniconda/Miniconda3-4.5.4-Linux-x86_64.sh -O miniconda.sh -nv
   bash miniconda.sh -b -p $HOME/miniconda
   export PATH="$HOME/miniconda/bin:$PATH"
-  pip install -q scipy tensorflow==$tf_version \
+  case "$(lsb_release -r -s || true)" in
```

Comment:
can you move this to `ci/travis/install-strace.sh` ?

Reply:
Done!